exercicios/para-casa/index.js

```diff
@@ -0,0 +1,14 @@
+const cafeteria = require ("./cafeteria.json")
+console.log(cafeteria)
+
+function exibirMenu() {
+    for (let i = 0; i < cafeteria.length; i++) {
```

Comment:
Quando for fazer iterações com listas eu sugiro usar essas duas formas abaixo, pois elas são mais simples de escrever, e estão menos propensas à erros, uma vez que você não precisa ficar elaborando uma codição lógica para a iteração seguir, como a que você faz em `i < cafeteria.length`.

```
cafeteria.forEach((item) => {
    //codigo aqui
});
```

ou

```
for (let item of cafeteria) {
    //codigo aqui
}
```